src/main/java/ua/lviv/iot/models/loom/Loom.java

```diff
@@ -0,0 +1,24 @@
+package ua.lviv.iot.models.loom;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data // @ToString, @EqualsAndHashCode, @Getter, @Setter, @RequiredArgsConstructor
+@NoArgsConstructor
+public abstract class Loom {
+
+    protected Country originCountry;
```

Comment:
why those fields are protected?

Reply:
I did it according to my UML diagram and I thought it would be good to have direct access to them in derived classes. Is it better to make them private?